tsconfig.json

```diff
@@ -1,125 +1,132 @@
-        "experimentalDecorators": true,
-        "emitDecoratorMetadata": true,
-        "lib": ["es6", "dom", "dom.iterable"],
-        "module": "commonjs",
-        "moduleResolution": "node",
-        "noEmit": false,
-        "sourceMap": true,
-        "strict": true,
-        "strictPropertyInitialization": false,
-        "target": "ES6",
-        "skipLibCheck": true,
-        "resolveJsonModule": true,
-        "useUnknownInCatchVariables": false
-    },
-    "references": [
-        {
-            "path": "packages/adp-tooling"
-        },
-        {
-            "path": "packages/app-config-writer"
-        },
-        {
-            "path": "packages/axios-extension"
-        },
-        {
-            "path": "packages/backend-proxy-middleware"
-        },
-        {
-            "path": "packages/btp-utils"
-        },
-        {
-            "path": "packages/cap-config-writer"
-        },
-        {
-            "path": "packages/control-property-editor-common"
-        },
-        {
-            "path": "packages/control-property-editor"
-        },
-        {
-            "path": "packages/create"
-        },
-        {
-            "path": "packages/deploy-tooling"
-        },
-        {
-            "path": "packages/environment-check"
-        },
-        {
-            "path": "packages/eslint-plugin-fiori-tools"
-        },
-        {
-            "path": "packages/fe-fpm-writer"
-        },
-        {
-            "path": "packages/fiori-elements-writer"
-        },
-        {
-            "path": "packages/fiori-freestyle-writer"
-        },
-        {
-            "path": "packages/logger"
-        },
-        {
-            "path": "packages/mockserver-config-writer"
-        },
-        {
-            "path": "packages/odata-service-writer"
-        },
-        {
-            "path": "packages/preview-middleware-client"
-        },
-        {
-            "path": "packages/preview-middleware"
-        },
-        {
-            "path": "packages/project-access"
-        },
-        {
-            "path": "packages/project-input-validator"
-        },
-        {
-            "path": "packages/store"
-        },
-        {
-            "path": "packages/system-access"
-        },
-        {
-            "path": "packages/ui-components"
-        },
-        {
-            "path": "packages/ui5-application-writer"
-        },
-        {
-            "path": "packages/ui5-config"
-        },
-        {
-            "path": "packages/ui5-info"
-        },
-        {
-            "path": "packages/ui5-library-writer"
-        },
-        {
-            "path": "packages/ui5-proxy-middleware"
-        },
-        {
-            "path": "packages/ui5-test-writer"
-        },
-        {
-            "path": "packages/yaml"
-        },
-        {
-            "path": "examples/odata-cli"
-        },
-        {
-            "path": "examples/simple-generator"
-        }
-    ]
+  "compilerOptions": {
```

Comment:
Don't we have 4 spaces indention?
See 
https://github.com/SAP/open-ux-tools/blob/8876afc10e00138769a168cac1f03b22d5d52830/.prettierrc.js#L22-L24

Reply:
Thanks